gneiss/stats/composition.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from skbio import DistanceMatrix
+from skbio.stats.composition import closure
+
+
+def variation_matrix(X):
+    """ Calculate Aitchison variation matrix.
+
+    This calculates the Aitchison variation matrix.  Given a compositional
+    matrix :math:`X`, row :math:`i` and column:math:`j`, in the variation
+    matrix of :math:`X` is given by
+
```

Comment:
Shouldn't i and j both point to two rows, i.e. "comparing" two samples instead of mixing one sample and one feature?